app-accessibility/caribou/files/caribou-fix-python-env.patch

```diff
@@ -0,0 +1,60 @@
+diff -rupN caribou-0.4.21.orig/bin/antler-keyboard.in caribou-0.4.21/bin/antler-keyboard.in
+--- a/bin/antler-keyboard.in	2016-07-20 01:13:11.000000000 -0400
++++ b/bin/antler-keyboard.in	2021-01-24 23:54:14.466466751 -0500
```

Comment:
Make the [preamble as simple as possible](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html):
```
--- a/bin/antler-keyboard.in
+++ b/bin/antler-keyboard.in
```